levelFeature/constants.py

```diff
@@ -1,2 +1,2 @@
 POST_LATEST = 'http://127.0.0.1:8000/post_reading/'
-DELETE_READINGS = 'http://127.0.0.1:8000/delete_reading/'
+DELETE_READINGS = 'http://127.0.0.1:8000/delete_readings/'
```

Comment:
update endpoint?

Reply:
Yes, at first only one reading was being deleted at a time, but I made it so that all readings are cleared at once, so I just updated this endpoint to make this more clear.

Reply:
I meant why 127.0.0.1:8000? Change it to AWS.

Reply:
Okay, I changed it.